### ISSUE TITLE
WebDriver Backed Selenium: isEditable is not available

### ISSUE THREAD

**Issue by alessiodm** [SeleniumHQ/selenium#3842]
## Meta -
OS:  Linux, OSX

Selenium Version: 2.44.0, but should be all 

Browser:  All

Browser Version:  Chromium 55.0.2883.95, but should be all

## Expected Behavior -
The `isEditable` command should be available.

## Actual Behavior -
We get a Selenium error: `selenium.isEditable is not a function`

## Steps to reproduce -

Setting the `chromedriver` path correctly, just run:
`$> groovy -Dwebdriver.chrome.driver=<your_path> IsEditableIssue.groovy`

Script:
```groovy
@Grapes([
    @Grab("org.seleniumhq.selenium:selenium-chrome-driver:2.44.0"),
    @Grab("org.seleniumhq.selenium:selenium-support:2.44.0"),
    @Grab("org.seleniumhq.selenium:selenium-java:2.44.0")
])
import org.openqa.selenium.*;
import org.openqa.selenium.chrome.*;
import com.thoughtworks.selenium.webdriven.*;

class IsEditableIssue {

    public static void main(String[] args) {
        ChromeOptions options = new ChromeOptions();
        options.addArguments("window-size=1024,768");
        ChromeDriver driver = new ChromeDriver(options);

        try {
            WebDriverCommandProcessor p = new WebDriverCommandProcessor('https://google.com', driver)
            p.doCommand('open', (String[]) [ '/' ])
            p.doCommand('waitForCondition', (String[]) [ 'selenium.isElementPresent("//input")', '5000' ])
            // Here we are going to have an exception
            p.doCommand('waitForCondition', (String[]) [ 'selenium.isEditable("//input")', '5000' ])
        }
        catch (Exception ex) {
            ex.printStackTrace()
            driver.quit()
        }
    }

}
```

Output:
```
Starting ChromeDriver 2.28.455517 (2c6d2707d8ea850c862f04ac066724273981e88f) on port 47301
Only local connections are allowed.
com.thoughtworks.selenium.SeleniumException: unknown error: selenium.isEditable is not a function
  (Session info: chrome=57.0.2987.133)
  (Driver info: chromedriver=2.28.455517 (2c6d2707d8ea850c862f04ac066724273981e88f),platform=Mac OS X 10.11.6 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 14 milliseconds
[...]
```

**Comment by alessiodm** [comment]
The `isChecked` method should suffer of the same issue as well.

**Comment by shs96c** [comment]
The RC APIs were retired when we shipped 3.0 over two years ago, and were deprecated for a long time before that. I'd be happy to merge a fix for this into the tree, but I'm not going to write the code myself.

Having said that, the fix is pretty simple, since you're using the `WebDriverCommandProcessor`: just unwrap the underlying webdriver (using `getWrappedDriver()`) and then use that to find the element and check the property or attribute you're interested in.

I'm closing this issue, but I'm serious about being happy to merge a PR that fixes this into the tree.
